client/homebrew/phbStyle/phb.style.less

```diff
@@ -468,3 +468,99 @@ body {
 		margin-bottom : 10px;
 	}
 }
+//*****************************
+// *      MAGIC ITEM CARD
+// *****************************/
+.phb .card {
+	width: var(--card-width);
+	height: var(--card-height);
+	-webkit-print-color-adjust:exact;
+	border: 2px solid black;
+	border-radius: 20px;
+	column-count: 1;
+	padding-left: 10px;
+	padding-right: 10px;
+	column-span: all;
```

Comment:
What about allowing cards to be put into separate columns? Here is an example of how it could look like: [share link](https://homebrewery.naturalcrit.com/share/b6Qj3zcgxg-Y)

Here I removed `column-span: all` by default, but it is being added if card is also marked with `.centered` class.

Also note change in margin: cards are too close if they are not separated manually by `<br>` and I think it would be a nice addition, because it removes the need in an explicit HTML-markup in the brew source.

One more note about columns: we should probably use something like `break-inside: avoid-column;` to avoid cards being broken when they exceed column limits: I've also have this change in the shared brew and a page with an example

Reply:
I like it - I was mostly approaching from the mindset of having both sides of the card; this is why I started with the "each page is a card face" approach. However now that it's a single face is confined to a `div` and can be included, this is a better approach.

Reply:
Also `break-inside: avoid` seems to work better than `avoid-column`, but this may just be an artifact of cross-browser compatibility (Firefox vs Chrome).